backend/src/configuration/swagger.ts

```diff
@@ -10,7 +10,7 @@ function initSwagger(): object {
         version: '1.0.0',
         description: 'This is BSA project.',
       },
-      host: CONFIG.APP.HOST,
+      host: CONFIG.APP.HOST === 'localhost' ? `localhost:${CONFIG.APP.PORT}` : CONFIG.APP.HOST,
```

Comment:
move this check to general config and use isDevEnvironment function to check if need to add the port number in HOST variable.